### PR TITLE
Autocue - Introducing sustained ending support

### PR DESCRIPTION
**Pull request by RM-FM**
This is an improvement of autocue to optimize crossfade and cue out points in case of sustained endings. This new algorithm is analysing the loudness behavior after the initially calculated crossfade and cue out point. If it's still a part of the song that is likely to be of relevance, the crossfade is recalculated based on that data.

The relevance is determined on the basis of the loudness reduction immediately after the original crossfade point and a possible "plateau" (i.e. no continuous fade-out).

### CODE REVIEW

**Comment by RM-FM** [comment]
@toots After some tweaking, I'm very happy with the result.

File: src/libs/autocue.liq [hunk 1]
```diff
@@ -468,9 +753,6 @@ def enable_autocue_metadata() =
   settings.crossfade.assume_autocue := true
 
   def autocue_metadata(~metadata, fname) =
-    print(
-      "fname: #{fname}, metadata: #{metadata}"
-    )
```
Comment (toots) [inline src/libs/autocue.liq:473]: Whoops I think that was me.

**Comment by RM-FM** [comment]
@toots I've completed my tests. Please feel free to test as well...
Default is `settings.autocue.internal.sustained_endings_enabled := true`.

**Comment by toots** [comment]
This has been integrated with #3870 and will be merged via that PR.

**Pull request closed without merging**